tf_bodypix/draw.py

```diff
@@ -41,14 +41,24 @@ def get_adjacent_keypoints(
 
 
 def get_cv_keypoints(keypoints: Iterable[Keypoint]) -> List[cv2.KeyPoint]:
-    return [
-        cv2.KeyPoint(
-            x=keypoint.position.x,
-            y=keypoint.position.y,
-            size=3
-        )
-        for keypoint in keypoints
-    ]
+    try:
+        return [
+            cv2.KeyPoint(
+                x=keypoint.position.x,
+                y=keypoint.position.y,
+                size=3
+            )
+            for keypoint in keypoints
+        ]
+    except Exception:
+        return [
```

Comment:
```suggestion
        # backwards compatibility with opencv 4.5.2 and below
        return [
```

The change to `size` was introduced in [4.5.3](https://docs.opencv.org/4.5.3/d2/d29/classcv_1_1KeyPoint.html), and was still `_size` in [4.5.2](https://docs.opencv.org/4.5.2/d2/d29/classcv_1_1KeyPoint.html).
It is good to have the comment here so it can be removed once 4.5.2 is no longer supported.